### ISSUE TITLE
Incompatible with 1.19.2

### ISSUE THREAD

**Issue by hexadecimal233** [shedaniel/slight-gui-modifications#83]
no mod installed
... Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method postRenderCommandSuggestor(Lnet/minecraft/class_4587;IIFLorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V in slight-gui-modifications.mixins.json:MixinChatScreen from mod slight-gui-modifications failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap slight-gui-modifications-refmap.json
...

**Comment by Disguys** [comment]
@shedaniel please fix complatible with 1.19.2
```
[20:09:18] [main/INFO]: Loading Minecraft 1.19.2 with Fabric Loader 0.14.9
[20:09:18] [main/WARN]: Mod resolution failed
[20:09:18] [main/INFO]: Immediate reason: [HARD_DEP_NO_CANDIDATE slight-gui-modifications 2.2.0 {depends cloth-client-events-v0 @ [>=4-]}, ROOT_FORCELOAD_SINGLE slight-gui-modifications 2.2.0]
[20:09:18] [main/INFO]: Reason: [HARD_DEP slight-gui-modifications 2.2.0 {depends cloth-client-events-v0 @ [>=4-]}]
[20:09:18] [main/INFO]: Fix: add [add:cloth-client-events-v0 4- ([[4-,?)])], remove [], replace []
[20:09:18] [main/ERROR]: Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Install cloth-client-events-v0, version 4- or later.
Unmet dependency listing:
	 - Mod ''Slight' GUI Modifications' (slight-gui-modifications) 2.2.0 requires version 4- or later of cloth-client-events-v0, which is missing!
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:190) ~[fabric-loader-0.14.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:148) ~[fabric-loader-0.14.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68) [fabric-loader-0.14.9.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.14.9.jar:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at org.polymc.impl.OneSixLauncher.invokeMain(OneSixLauncher.java:104) [NewLaunch.jar:?]
	at org.polymc.impl.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:175) [NewLaunch.jar:?]
	at org.polymc.impl.OneSixLauncher.launch(OneSixLauncher.java:185) [NewLaunch.jar:?]
	at org.polymc.EntryPoint.listen(EntryPoint.java:144) [NewLaunch.jar:?]
	at org.polymc.EntryPoint.main(EntryPoint.java:74) [NewLaunch.jar:?]

**Comment by shedaniel** [comment]
please stop ping it will be ported when its ported
